src/cdn_definitions/schema.yaml

```diff
@@ -11,7 +11,170 @@ $id: http://release-engineering.github.io/cdn-definitions/schema.json
 
 
 definitions:
+  arch:
+    type: string
+
+  arch_list:
+    type: array
+    items:
+      $ref: "#/definitions/arch"
+    uniqueItems: true
+
+  release_version:
+    type: string
+    pattern: "^[0-9]+\\.[0-9]+$"
+
+  major_version:
+    type: integer
+
+  major_version_string:
+    type: string
+    pattern: "^[0-9]+$"
+
+  signing_key:
+    type: string
+    pattern: "^[A-F0-9]{8}"
```

Comment:
Just noticed this is missing the ending anchor "$", so after the initial 8 characters it would accept any old garbage.